manifest.json

```diff
@@ -63,7 +63,7 @@
           "angular-aria.min.js", "angular-aria.min.js.map"
         ],
         "jsFiles": [
```

Comment:
I know we touched a bit on this before, but it seems odd to me that "files" has such different semantics here from "jsFiles" and "cssFiles". What are your thoughts on calling the last two something like "jsFilesToBundle" or "jsFilesToMinify" instead -- or alternatively doing something like the following:

    "bundles": {
      "js": [...],
      "css": [...]
    }

I rather like the latter, but would be interested in what you think.